common/common.scss

```diff
@@ -37,3 +37,17 @@
     }
   }
 }
+
+@if $min_user_level == user {
```

Comment:
Don't think it's a good idea to had conditions in the scss files (if it even works?)

Reply:
dunno, but there is already condition for `left`.

It's quite simple, it just hides the whole block if the staff/user class is present.

Reply:
We already have `anon` and `staff` class on the `body` / `html` elements. Why not use that instead?

Reply:
I think that's exactly what I did?

Reply:
Right... Then why the need for the `@if` ?

Reply:
It's a component option that specifies when the links should be displayed. Very similar to the position option.